packages/mobile/locales/en-US/global.json

```diff
@@ -122,6 +122,7 @@
   "addressLookupFailure": "Failed to look up phone number address",
   "saltFetchFailure": "Failed to retrieve phone number secret",
   "saltQuotaExceededError": "Your phone number lookup quota is used up",
+  "odisInsufficientBalance": "Your balance is too low to make this transaction",
```

Comment:
nit: "Your balance is too low to complete this process" would be a bit more user friendly